Lib/pdb.py

```diff
@@ -2232,7 +2232,7 @@ def main():
             print("The program exited via sys.exit(). Exit status:", end=' ')
             print(e)
         except BaseException as e:
-            traceback.print_exc()
+            traceback._print_exception_bltin(e)
```

Comment:
> We can also use `print_exception` but we need `traceback._can_colorize()` anyway - which should be put into a separate module if we want to colorize more modules.

`_colorize` is now its own module. Let's change this:

```suggestion
            traceback._print_exception_bltin(e, colorize=_colorize.can_colorize())
```

Reply:
Just double checked, `_print_exception_bltin` only takes an exception. It queries `_colorize` inside the function.

Reply:
Oh you want me to switch to `print_exception` the public interface?